src/services/ExecutionService.js

```diff
@@ -0,0 +1,12 @@
+import { Op } from "sequelize"
+import { ExecutionDAO } from "../models"
+
+export default class ExecutionService {
+
+  async createExecution(payload) {
+    payload.executions.forEach(function(itm){
+      itm.deviceToken = payload.deviceToken;
+     });
+      await ExecutionDAO.bulkCreate(payload.executions)
+  }
+}
```

Comment:
Só adicionar uma nova linha no fim do arquivo

```suggestion
}

```